src/Main.elm

```diff
@@ -110,10 +140,26 @@ view address model =
                     ]
                 ]
                 <| viewList model
+            , div
+                [ style
+                    [ ( "display", "flex" )
+                    , ( "flex-direction", "row" )
+                    ]
+                ]
+                [ pageButton address Prev "Newer"
+                , pageButton address Next "Older"
+                ]
             ]
         ]
 
 
+pageButton : Address a -> a -> String -> Html
+pageButton address action label =
+    button
+        [ onClick address action ]
```

Comment:
Can we include a `disabled` attribute on the button based on whether the user is already at the beginning or end of the list? You can extract the logic from the conditionals in `update`.